penguin/cli/cli.py

```
@@ -795,6 +799,94 @@ def _manual_skill_install_message() -> str:
     )
 
 
+
+def _mcp_tool_manager() -> Any:
```

Ruff (0.15.12)

[warning] Dynamically typed expressions (typing.Any) are disallowed in `_mcp_tool_manager`

(ANN401)